src/Symfony/Messenger/Processor.php

```diff
@@ -54,7 +54,7 @@ private function remove(mixed $data): void
         );
     }
 
-    public function process(mixed $data, Operation $operation, array $uriVariables = [], array $context = []): mixed
+    public function process(object $data, Operation $operation, array $uriVariables = [], array $context = []): ?object
```

Comment:
To be consistent with `src/Doctrine/Common/State/RemoveProcessor.php`, you should return null in case of DeleteOperationInterface.